cfpb/cfgov-refresh
# Security fix for using data-attribute and converting to html

Security fix for using data-attribute and converting to html

## Changes

- Modified `cfgov/legacy/static/nemo/_/js/functions.js` to escape data attribute content before inserting into html. 
- Modified `cfgov/v1/static-legacy/js/functions.js` to escape data attribute content before inserting into html. 

## Testing

1. Visit `http://localhost:8000/find-a-housing-counselor/?zipcode=22304` and  view source.
2. Click the `view Paperwork Reduction Act statement`link. 
3. Verify that the link text toggles, when the link is clicked. 


## Checklist
